docs/themes/thxvscode/layouts/partials/breadcrumb.html

```diff
@@ -62,10 +62,10 @@
   {{ if $packageDisplayName }}
     <p>
       <a href="/docs/api/v1" onclick="return false;">{{ humanize $concept | title }}</a> <!-- TODO: remove onclick and create proper link to new concept readme -->
-        {{ $packageLink := print "/docs/api/v1/" $packageName "/" }}
+        {{ $packageLink := print "/docs/apis/" $packageName "/" }}
```

Comment:
It is probably best to put this back to `apis` for now, but this isn't really correct either. This needs to point to `docs/api/<specified-version-string>`, otherwise the breadcrumb will always link to the "current" version even when the user is looking at docs for an earlier version.

We should file a task to fix this, probably. We'll want to update this template to take in a version string parameter and use that.

Reply:
Updated to be /api/<selectedVersion>